go.mod

```diff
@@ -7,13 +7,15 @@ require (
 	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
 	github.com/alecthomas/units v0.0.0-20210912230133-d1bdfacee922 // indirect
 	github.com/golang/protobuf v1.3.3
+	github.com/google/addlicense v1.0.0 // indirect
```

Comment:
Can you pls revert these changes? This package is actually not needed, we only use it for the CI run